server/routes/auth.js

```diff
@@ -18,35 +18,23 @@ router.post(
 
 // post -> signup
 router.post(
-  '/signup',
+  "/signup",
   userController.createUser,
   sessionController.startSession,
   (req, res, next) => {
-    if (res.locals.user.is_landlord) {
-      landlordController.postLandlord(req, res, next);
```

Comment:
see previous comment regarding the separation of responsibilities between `landlordController` and `userController`. with modulated code, we would have the landlord creation separate from the user creation.

Reply:
also, feel free to revise the structure of the middleware calls after considering the above mentioned changes

Reply:
fixed!